go.mod

```diff
@@ -86,3 +86,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
+
+replace github.com/aquasecurity/tracee/types v0.0.0-20220412160215-ccc81e5e59cd => github.com/roikol/tracee/types v0.0.0-20220516100414-ca8aafbe6202
```

Comment:
Was this left here by mistake?

Reply:
nope, it was needed so the code will compile. since the PR to change the types (https://github.com/aquasecurity/tracee/pull/1762) was not merged when i opened this PR.
now the PR to change the types is merged, so no need for the `replace` anymore.